internal/backuprestore/backup.go

```diff
@@ -254,6 +254,70 @@ func (h *BRHandler) createNewBackupCr(ctx context.Context, backup *velerov1.Back
 	return nil
 }
 
+// +kubebuilder:rbac:groups=core,resources=persistentvolumes,verbs=get;list;watch;update
+
+// PatchPVsReclaimPolicy - when LVMS is configured, we need to patch PVs with Retain as
+// the persistentVolumeReclaimPolicy due to https://github.com/vmware-tanzu/velero/issues/2739.
+func (h *BRHandler) PatchPVsReclaimPolicy(ctx context.Context) error {
+	pvList := &corev1.PersistentVolumeList{}
+	if err := h.List(ctx, pvList); err != nil {
+		return fmt.Errorf("failed to list PersistentVolumes: %w", err)
+	}
+
+	if len(pvList.Items) == 0 {
+		h.Log.Info("No PersistentVolumes found in the cluster, skipping.")
+		return nil
+	}
+
+	for _, pv := range pvList.Items {
+		hasLvmsAnnotation := pv.GetAnnotations()[topolvmAnnotation] == topolvmValue
```

Comment:
The annotations could be nil if pv is manually created with `oc create -f  `.
It's safer to add check `if pv.GetAnnotations() != nil`

Reply:
nice, thanks